metricbeat/module/apache/status/data.go

```diff
@@ -16,41 +17,41 @@ var (
 	// This should match: "CPUSystem: .01"
 	matchNumber = regexp.MustCompile("(^[0-9a-zA-Z ]+):\\s+(\\d*\\.?\\d+)")
 
-	schema = h.NewSchema(common.MapStr{
-		"total_accesses":    h.Int("Total Accesses"),
-		"total_kbytes":      h.Int("Total kBytes"),
-		"requests_per_sec":  h.Float("ReqPerSec", h.Optional),
-		"bytes_per_sec":     h.Float("BytesPerSec", h.Optional),
-		"bytes_per_request": h.Float("BytesPerReq", h.Optional),
-		"workers": common.MapStr{
-			"busy": h.Int("BusyWorkers"),
-			"idle": h.Int("IdleWorkers"),
+	schema_ = schema.Schema{
```

Comment:
Little bit odd naming convention, especially as it will also be used by others. Perhaps going with `s` and using `c` for the s. C would stand for conversion. BTW: if we use `.` for the include `mapstrstr`. Never tried it but that should make it part of the same namespace. So we could use it without prefix.